### PR TITLE
Aaronthetechguy

### PR DESCRIPTION
**Pull request by Aaronthetechguy**
fixed Tomcat launch on macOS

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ spring:
     timeBetweenEvictionRunsMillis: 25200000 #7*60*60*1000
     validationQuery: SELECT 1
   jpa:
+    database-platform: org.hibernate.dialect.MySQL5Dialect
```
Comment (marod424) [inline src/main/resources/application.yml:14]: Great job with this fix!

**Comment by marod424** [comment]
Notice that the pull request has merge conflicts. This is a result in differences in the `.idea/workspace.xml` file. The good news is in the trunk the `.idea` directory has been added to `.gitignore` (since it's Intellij IDE related) so this will help us avoid these merge conflicts in the future.

We could just resolve the conflicts here, and then merge the pull request, however I would encourage a different route. I suggest that you merge the latest trunk into your local branch and push those changes to your remote branch. That should resolve the merge conflict and then this overall PR will only include the changes to the `application.yml` file. Further, it is something you will want to do regardless with the trunk and your branch since it's good to have the latest changes. I think it would be a great demo in class today if you are up for it.

**Comment by Aaronthetechguy** [comment]
I actually have no idea how to do that so you’re going to have to show me

**Comment by marod424** [comment]
Of course, it will make for a good in class lesson today.

**Review by marod424 — APPROVED** [review]

**Pull request merged**